src/utils/analytics-helpers.ts

```
@@ -533,22 +759,193 @@ export class AnalyticsCalculator {
-      case 'resolution_time':
-        const resolved = disputes.filter(d => d.resolutionTime);
-        return resolved.length > 0
-          ? resolved.reduce((sum, d) => sum + (d.resolutionTime || 0), 0) / resolved.length
-          : 0;
-      case 'satisfaction':
-        const withSatisfaction = disputes.filter(d => d.userSatisfactionScore);
-        return withSatisfaction.length > 0
-          ? withSatisfaction.reduce((sum, d) => sum + (d.userSatisfactionScore || 0), 0) / withSatisfaction.length
+        return applications.length;
+      case 'success_rate':
+        const successful = applications.filter(app => app.status === ApplicationStatus.ACCEPTED);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)